src/pages/ResourcePresentation.tsx

```diff
@@ -120,11 +121,17 @@ const ResourcePresentation: FC<ResourcePresentationProps> = ({ resource }) => {
           </StyledFeatureWrapper>
         )}
 
-        {resource.licenses && resource.licenses.length !== 0 && (
+        {resource.licenses && resource.licenses.length !== 0 && resource.licenses[0].identifier.length > 0 && (
           <StyledFeatureWrapper>
             <StyledCaption variant="caption">{t('resource.metadata.license')}</StyledCaption>
             {resource.licenses.map((license) => (
-              <LicenseCard key={license.identifier} license={license} />
+              <>
```

Comment:
bør ikke trenge fragment

Reply:
SyntaxError: C:\Users\ansi\IdeaProjects\dlr-gui-new\src\pages\ResourcePresentation.tsx: Unexpected token, expected "," (129:24)

![image](https://user-images.githubusercontent.com/44363838/102594914-24a86100-4117-11eb-9d5b-1aec472464a7.png)

Reply:
et sett med paranteser for mye ?

Reply:
ok. man må inn med en return i tilfelle. Da er fragment bedre læll :-)

Reply:
Ahh, supert. Fikset